bin/create-gem.py

```diff
@@ -54,7 +54,7 @@
 
   # Get the sample name from the file name.
   file_basename = os.path.basename(result)
-  sample_name = file_basename.split('_vs_')[0]
+  sample_name = file_basename.split('.')[0]
```

Comment:
Hey John, why this change?

Reply:
I didn't want the user to have to input a reference name if they were using Kallisto or Salmon. I thought that it would be confusing. I also thought that having this additional "_vs_" doesn't really make since to have. The user knows what they are aligning to, and including this information in the title seemed redundant.

Reply:
I think we need to keep the _vs_{reference}.  If someone runs the same samples against different reference versions and decides to keep and share the BAM and other output files then it can get confusing which reference the reads were aligned to.

Reply:
Alternatively, if we follow my suggestion to get rid of the reference_prefix then we would use the machine_name for the project rather than the `_vs_{reference}`.  That I think would do the same thing but if we do that we wouldn't need to use the `_vs_` string.